packages/core/src/App/Containers/ResetEmailModal/confirm-email-modal.jsx

```diff
@@ -0,0 +1,118 @@
+
+        WS.changeEmail(api_request).then(response => {
+            if (response.error) {
+                setIsOpen(false);
+                onClose(true);
+                set_is_error(true);
+                set_error_message(response.error.message);
+                pullErrorMessage(response.error.message);
+            } else {
+                setIsSendEmailModalOpen(true);
+                setIsOpen(false);
+            }
+        });
+    };
+
+    const resendEmail = () => {
+        WS.changeEmail(email_request);
+    };
+
+    if (is_send_email_modal_open) {
+        return (
+            <SentEmailModal
+                is_open={is_send_email_modal_open}
+                onClose={() => setIsSendEmailModalOpen(false)}
+                identifier_title={'Change_Email'}
+                onClickSendEmail={resendEmail}
+                has_live_chat={true}
+                is_modal_when_mobile={true}
+            />
+        );
+    }
+    return (
+        <Modal
+            is_open={isOpen}
+            should_header_stick_body
+            title={<Localize i18n_default_text='Are you sure?' />}
+            toggleModal={onClose}
+            width='440px'
+        >
+            {is_error ? <p>{error_message}</p> : null}
+            <React.Fragment>
+                <div className='email-confirmation'>
+                    <Text as='p' color='prominent' size='xs' align='left'>
+                        <Localize
+                            i18n_default_text='Are you sure you want to update email <0>{{prevEmail}}</0> to <1>{{newEmail}}</1>?'
+                            values={{ newEmail, prevEmail }}
+                            components={[
+                                <span key={0} className='email-confirmation__prevEmail' />,
+                                <span key={1} className='email-confirmation__currentEmail' />,
+                            ]}
+                        />
+                    </Text>
+                </div>
+
+                <Modal.Footer>
+                    <Button onClick={onClose} has_effect text={localize('Cancel')} secondary large />
+                    <Button
+                        className='email-change_button'
+                        has_effect
+                        onClick={() => {
+                            handleSubmit();
+                        }}
+                        primary
+                        large
+                    >
+                        <Localize i18n_default_text='Confirm' />
+                    </Button>
+                </Modal.Footer>
+            </React.Fragment>
+        </Modal>
+    );
+};
+
+ConfirmEmailModal.propTypes = {
+    email: PropTypes.string,
+    disableApp: PropTypes.func,
+    enableApp: PropTypes.func,
+    onClose: PropTypes.func,
+    is_open: PropTypes.bool,
+    is_loading: PropTypes.bool,
+    is_visible: PropTypes.bool,
+    logoutClient: PropTypes.func,
+    verification_code: PropTypes.string,
+};
```

Comment:
ConfirmEmailModal is only taking onClose, is_open, prevEmail, newEmail, verification_code, pullErrorMessage as props. Can we remove unwanted props?